<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSelectStartFolder">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Start Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelectedStartFolder">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Folder: none</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartProcessing">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Processing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter file name mask:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputFileMask">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectOutFilePath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Selet Output Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelectedOutFolder">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Folder: none</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRewrite">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>What to do if output file exist: checked - rewrite, unchecked - add counter to new file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHexMask">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0123456789abcdef</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBitMask">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;8-byte bit mask - 16 hex symbols starting with NO leading 0x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTimer">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>ms</string>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRepeat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check this box of you want the app run processing by timer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
